website/pages/docs/configuration/audit.mdx

```diff
@@ -293,4 +293,10 @@ If the request returns an error the audit log will reflect the error message.
 }
 ```
 
+### `audit` Configuration Reloads
```

Comment:
It seems like enable/disable would only be supported, but it also looks like there's a bug so it currently does not work :( Will file an issue.